### PR TITLE
Increased coverge for github management commands files

### LINKED ISSUE #182: Increase backend github/management coverage
We don't have any coverage here and want it to be 75%+

```
apps/github/management/commands/github_enrich_issues.py                            38     38      8      0     0%   3-62
apps/github/management/commands/github_update_owasp_organization.py                64     64     16      0     0%   3-116
apps/github/management/commands/github_update_project_related_repositories.py      46     46     10      0     0%   3-64
```

### PR DESCRIPTION
**Pull request by Naveen-Pal**
Resolves #182 

I have added the test file for the github/management/commands files. I have only tested the add_argument function. As handle function have some issue as it was connecting to database ( in this code "Issue.bulk_save(issues)"  ). I will again check that tomorrow or your suggestion may help. 
<br/>
In this pull request,
Total coverage have increased to 40%
<br/>
image
<br/>
image

### CODE REVIEW

**Comment by arkid15r** [comment]
It's a good start and great improvement. In order to get the issue resolved we need 75%+ for these files.
I'll wait for the rest of test and then review it. Marking as draft for now.

**Comment by Naveen-Pal** [comment]
I have updated the test to make their coverage 75%+
<br/>
image
<br/>
image

File: backend/tests/github/management/commands/github_enrich_issues_tests.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    mock_ordered_queryset.__iter__.return_value = iter(mock_issues)
+    mock_ordered_queryset.count.return_value = len(mock_issues)
+    mock_ordered_queryset.__getitem__ = (
+        lambda _, idx: mock_issues[idx]
+        if isinstance(idx, int)
+        else mock_issues[idx.start : idx.stop]
+    )
+
+    type(mock_issue_class).open_issues = PropertyMock(return_value=mock_open_issues)
+    type(mock_open_issues).without_summary = PropertyMock(return_value=mock_open_issues)
```
Comment (arkid15r) [inline backend/tests/github/management/commands/github_enrich_issues_tests.py]: Is it possible to use other (more readable) syntax here?
If not please add a comment about `type(` usage

File: backend/tests/github/management/commands/github_update_owasp_organization_tests.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        Project, "objects"
+    ) as mock_project_objects, mock.patch.object(
+        Chapter, "objects"
+    ) as mock_chapter_objects, mock.patch.object(
+        Committee, "objects"
+    ) as mock_committee_objects, mock.patch.object(
+        Event, "objects"
+    ) as mock_event_objects, mock.patch.object(
+        Repository, "objects"
+    ) as mock_repository_objects, mock.patch("builtins.print") as mock_print:
```
Comment (arkid15r) [inline backend/tests/github/management/commands/github_update_owasp_organization_tests.py]: Could you format it in a more readable way using `with ():`

File: backend/tests/github/management/commands/github_update_owasp_organization_tests.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                assert mock_committee_update.call_count == expected_calls["committee"]
+            elif repository_name.startswith("www-event-"):
+                assert mock_event_update.call_count == expected_calls["event"]
+
+        mock_project_bulk_save.assert_called_once()
+        mock_chapter_bulk_save.assert_called_once()
+        mock_committee_bulk_save.assert_called_once()
+        mock_event_bulk_save.assert_called_once()
+
+        if not repository_name:
```
Comment (arkid15r) [inline backend/tests/github/management/commands/github_update_owasp_organization_tests.py]: Use `else` and move above the assertions.

File: backend/tests/github/management/commands/github_update_project_related_repositories_tests.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        Project, "bulk_save"
+    ) as mock_project_bulk_save, mock.patch("builtins.print") as mock_print:
+        command.handle(offset=offset)
+
+        mock_github.assert_called_once_with("fake-token", per_page=GITHUB_ITEMS_PER_PAGE)
+
+        if offset == 0:
+            mock_get_repository_path.assert_called_with("https://github.com/OWASP/test-repo")
+
+        if offset == 0:
```
Comment (arkid15r) [inline backend/tests/github/management/commands/github_update_project_related_repositories_tests.py]: Why 2 same ifs?

**Review by arkid15r — CHANGES_REQUESTED** [review]
Please look into these comments:

**Comment by Naveen-Pal** [comment]
@arkid15r 
please check and suggest if any changes required.

**Review by arkid15r — APPROVED** [review]
It looks much better, thank you @Naveen-Pal!

I'll take it!

**Pull request merged**